de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/communication/extensions/ProjectNegotiationOfferingExtension.java

```diff
@@ -14,19 +14,19 @@
 
     public static final Provider PROVIDER = new Provider();
 
-    private List<ProjectNegotiationData> projectNegotiationData;
+    private List<ReferencePointNegotiationData> projectNegotiationData;
     private TransferType transferType;
 
     public ProjectNegotiationOfferingExtension(String sessionID,
         String negotiationID,
-        List<ProjectNegotiationData> projectNegotiationData,
+        List<ReferencePointNegotiationData> projectNegotiationData,
```

Comment:
If the name is changed, you should change the variable names as well. (This refers to many more locations.)

Also you might want to think about abbreviations of "ReferencePoint" to RefPoint for brevity.